runtime/near-vm-errors/src/lib.rs

```diff
@@ -67,12 +85,12 @@ pub enum WasmTrap {
     IllegalArithmetic,
     /// Misaligned atomic access trap.
     MisalignedAtomicAccess,
-    /// Breakpoint trap.
-    BreakpointTrap,
     /// Stack overflow.
     StackOverflow,
     /// Generic trap.
     GenericTrap,
+    /// Indirect call to null.
+    IndirectCallToNull,
```

Comment:
Let's shift this error to where (impossible)  `BreakpointTrap` was, to avoid changing (serialized) error codes for overflow and generic trap.

Reply:
Done.